planet_scope/cloudless_mosaic/script.js

```diff
@@ -0,0 +1,66 @@
+//VERSION=3
+//Cloudless Mosaic with PlanetScope
+
+function setup() {
+  return {
+    input: ["red", "green", "blue", "dataMask", "clear"],
+    output: { bands: 4 },
+    mosaicking: "ORBIT"
+  };
+}
+
+function preProcessScenes(collections) {
+  collections.scenes.orbits = collections.scenes.orbits.filter(function (orbit) {
+    var orbitDateFrom = new Date(orbit.dateFrom);
+    return orbitDateFrom.getTime() >= (collections.to.getTime() - 7 * 24 * 3600 * 1000);
```

Comment:
Might be nice to expose the number of days to take into account as a separate variable

```suggestion
    const nAggregationDays = 7;
    return orbitDateFrom.getTime() >= (collections.to.getTime() - nAggregationDays * 24 * 3600 * 1000);
```